packages/primitives-core/src/styled.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react'
+import {
+  channel,
+  contextTypes,
+  setTheme,
+  testAlwaysTrue,
+  pickAssign,
+  interleave
+} from './utils'
+import { createCss } from './css'
+
+type State = {
+  theme: Object
+} | null
+
+let defaultPickTest = prop => prop !== 'theme' && prop !== 'innerRef'
+
+export function createStyled(
+  StyleSheet: Object,
+  getShouldForwardProp: Function = (cmp: React.ElementType): Function =>
```

Comment:
This should be `getShouldForwardProp: (cmp: React.ElementType) => (prop: string) => boolean = () => defaultPickTest` and could you make this argument into an object so that if there needs to be more options in the future it's easier?

Reply:
Yup! I'll fix it